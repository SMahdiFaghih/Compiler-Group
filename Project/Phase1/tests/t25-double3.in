25.000000001E+5
0.e455555555555
999999999999999.0000000000E0
999999999999999.0000000000E+0
999999999999999.0000000000E-0
