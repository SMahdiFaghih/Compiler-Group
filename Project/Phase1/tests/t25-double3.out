T_DOUBLELITERAL 25.000000001E+5
T_DOUBLELITERAL 0.e455555555555
T_DOUBLELITERAL 999999999999999.0000000000E0
T_DOUBLELITERAL 999999999999999.0000000000E+0
T_DOUBLELITERAL 999999999999999.0000000000E-0
